internal/scylla/client.go

```diff
@@ -280,3 +250,33 @@ func (c *Client) findAndSaveAccountID(ctx context.Context) error {
 
 	return nil
 }
+
+func getErrRetryInfo(err error) (retryable bool, retryAfter time.Duration) {
+	if oe, ok := err.(*net.OpError); ok {
+		return oe.Temporary(), 0
+	}
+
+	if apiErr := castToAPIError(err); apiErr != nil {
+		return apiErr.Temporary(), apiErr.RetryAfter
+	}
+	return false, 0
+}
+
+func sleepUntilCanceled(ctx context.Context, timeToSleep time.Duration) error {
```

Comment:
IMO this is not `UntilCanceled` since this func will return after `timeToSleep` has elapsed

Reply:
I have renamed it `sleep`, let me know if you have better name in mind